garage/tf/policies/gaussian_gru_policy.py

```diff
@@ -173,8 +184,8 @@ def __init__(
             LayersPowered.__init__(self, out_layers)
 
     @overrides
-    def dist_info_sym(self, obs_var, state_info_vars, name="dist_info_sym"):
-        with enclosing_scope(self.name, name):
+    def dist_info_sym(self, obs_var, state_info_vars, name=None):
+        with tf.name_scope(name, "dist_info", [obs_var, state_info_vars]):
```

Comment:
"dist_info_sym"?

Reply:
Well, the sym mean symbolic, that is, the function builds the graph of operations that will be executed later. I think adding sym to the scope name is redundant, since the scope basically only encapsulates tensor operations and variables that are symbolic.

Reply:
We will remove the _sym suffixes later when we make everything a _sym operation. For now, the scope should match the function name so it's easy to figure out which function produced which graph element.

Reply:
Got it, I completed all the scope names that were under symbolic functions.